python-checks/src/main/java/org/sonar/python/checks/TooManyParametersCheck.java

```diff
@@ -87,7 +87,7 @@ private void checkFunctionDef(SubscriptionContext ctx) {
 
   private boolean isAlreadyReportedInParent(FunctionSymbol functionSymbol) {
     // If the rule would already raise an issue on a parent class, don't raise the issue twice
-    return SymbolUtils.getOverriddenMethod(functionSymbol).map(f -> {
+    return SymbolUtils.getOverriddenMethods(functionSymbol).stream().findFirst().map(f -> {
```

Comment:
Here the change of behavior is okay because we might just remove FPs thanks to the fact `getOverriddenMethods` might return more than its counterpart, but I'm not sure it's really intentional, and it could have been different in another rule.